### PR TITLE
[xc admin] Pyth parser

### PR DESCRIPTION
**Pull request by guibescos**
Add PythMultisigInstruction and tests.

### CODE REVIEW

File: xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        deserializedData.name,
+        deserializedData.data,
+        resolveAccountNames(pythIdl as Idl, deserializedData.name, instruction)
+      );
+    } else {
+      return new PythMultisigInstruction(
+        "Unrecognized instruction",
+        {},
+        { named: {}, remaining: instruction.keys }
+      );
```
Comment (jayantk) [inline xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts:41]: you should think carefully about how you want to handle this case, as the wrong behavior for unknown instructions could cause you a lot of agony downstream.

is the right behavior to return something like `undefined`? Or should there be a specific instruction value that represents something unknown? Or should you have a default wrapper around TransactionInstruction that represents an unknown instruction (but retains all data in that instruction)?

Couple questions to ask yourself:
1. What will the users of PythMultisigInstruction want to do with an unknown instruction?
2. Do you expect MultisigInstruction to be convertible back into TransactionInstruction?

**Review by jayantk — APPROVED** [review]
lgtm, though see inline comment and think about it

File: xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { MultisigInstruction, MultisigInstructionProgram } from ".";
+import { AnchorAccounts, resolveAccountNames } from "./anchor";
+import { pythIdl, pythOracleCoder } from "@pythnetwork/client";
+import { TransactionInstruction } from "@solana/web3.js";
+import { Idl } from "@coral-xyz/anchor";
+
+export class PythMultisigInstruction implements MultisigInstruction {
```
[the remark this answers is not in the document]
Comment (guibescos) [reply xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts:7]: Future PR

File: xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts (hunk 1 above)
Comment (guibescos) [reply xc-admin/packages/xc-admin-common/src/multisig_transaction/PythMultisigInstruction.ts:41]: Future PR

**Pull request merged**